README.md

```diff
@@ -515,4 +520,61 @@ type FooType = {};
 type foo = {};
 ```
 
+## Rules (suppressing)
```

Comment:
This file is generated from the files in https://github.com/gajus/eslint-plugin-flowtype/tree/master/.README. You need to create a file for the rule here https://github.com/gajus/eslint-plugin-flowtype/tree/master/.README/rules and then include the file into the main template here https://raw.githubusercontent.com/gajus/eslint-plugin-flowtype/master/.README/README.md. Then you need to run [`npm run readme`](https://github.com/gajus/eslint-plugin-flowtype/blob/733e0761025a6835e9732703bb026af816a72649/package.json#L32).